### PR TITLE
extend line chart to support multiple datasets

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

File: lib/data/imports-definition.json [hunk 1]
```diff
@@ -69,12 +69,13 @@
...
       "name": "Time taken to clear permits",
-      "type": "bar",
+      "type": "line",
       "size": "medium",
       "units": "s",
       "displayRoundedData" : true,
+      "is_hero": true,
       "definition": "Time to clear measures the time between when a good requiring an import permit was held for assessment until the time it was cleared.",
-      "datasets": ["clearing-time"]
+      "datasets": ["clearing-time-2015","clearing-time-2016"]
```
Comment (elisechant) [inline lib/data/imports-definition.json]: if multiple datasets draw multiple lines on the chart

File: lib/data/imports-definition.json [hunk 2]
```diff
@@ -69,12 +69,13 @@
       "id":   "clearing-time",
       "updated_at": "2016-12-7T01:01:01.111Z",
       "name": "Time taken to clear permits",
-      "type": "bar",
+      "type": "line",
```
Comment (elisechant) [inline lib/data/imports-definition.json]: only make this permutation possible at this stage for line, but technically bar and others should also be possible

**Comment by elisechant** [comment]
the hard thing is getting all the labels to match the way we want

File: lib/data/imports-data.json [hunk 3]
```diff
@@ -51,13 +51,29 @@
...
-        "id":           "clearing-time",
+        "id":           "clearing-time-2015",
         "recorded_at":  "2016-09-07T01:01:01.111Z",
         "units":        "i",
         "name":         "Time to clear",
-        "note":         "This shows the average time taken to clear a permit using the Permit Verifcation Service.",
+        "note":         "This shows the average time taken to clear a permit using the Permit Verification Service.",
         "data": [
-          { "label": "2016-05", "value": 84807},
+          { "label": "2015-05", "value": 92940},
```
Comment (elisechant) [inline lib/data/imports-data.json]: label here currently *must* be a valid date or error

**Comment by elisechant** [comment]
this currently produces this chart:
screen shot 2017-01-10 at 11 47 26 am

File: lib/data/imports-data.json [hunk 4]
```diff
@@ -51,13 +51,29 @@
         ]
     },
     {
-        "id":           "clearing-time",
+        "id":           "clearing-time-2015",
```
Comment (micapam) [inline lib/data/imports-data.json]: Why break down by year ?

**Comment by micapam** [comment]
I'm confused as I don't see any code changes? Only data changes?

File: lib/data/imports-data.json (hunk 4 above)
Comment (elisechant) [inline lib/data/imports-data.json]: id's need to be unique

**Comment by elisechant** [comment]
@micapam that's the point. i just want to swarm on the data first

**Comment by elisechant** [comment]
trying to build this: 
screen shot 2017-01-10 at 11 52 10 am

**Pull request closed without merging**